### DISCUSSION TITLE
Where has the search button gone?

### DISCUSSION THREAD

**Discussion by PatAbah** [Acode-Foundation/Acode#624] [General]
![IMG_20230120_061925](https://user-images.githubusercontent.com/34959120/213624494-054f6e6d-9e9d-4432-beeb-1b05978d3e7b.jpg)
After updating the app, I see that my search button has been replaced by an "ESC" button (by the way what's the use of the ESC button again?)

I checked settings and there's apparently no way to replace the less useful "ESC" button with my good old search (I maintain large codes with Acode, and the quick search button is one of my most used buttons!)

I hope we'll be getting back the Search in subsequent versions (or perhaps a patch?)

**Comment by morteza-ayashi** [comment]
Hello, you can find it here
![Screenshot_20230121-183615_Acode.jpg](https://user-images.githubusercontent.com/113019884/213873178-5b67209c-1a8d-4ff4-8b23-c6461e9c9590.jpg)
![Screenshot_20230121-183621_Acode.jpg](https://user-images.githubusercontent.com/113019884/213873181-89e26ee4-fd09-4bce-becb-103728d03cb8.jpg)

I hope they move it to it's old good place

**Comment by PatAbah** [comment]
It's back! 😁👍
1.7.2 (268)

**Reply by morteza-ayashi** [reply]
Yes
